src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -27,12 +28,14 @@
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
+            + "[" + PREFIX_FAVORITE + "FAVORITE] "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_FAVORITE + "yes "
```

Comment:
just wondering from a user point fo view whether or not it's clunky for me to have to type `f/ yes` - should the presence of the `f/` tag already be sufficient to show that this contact should be favourited?

Reply:
Agreed, implemented! 👍